Source/Atomic/UI/UIScrollContainer.h

```diff
@@ -63,6 +63,8 @@ class UIScrollContainer : public UIWidget
     void SetAdaptContentSize(bool adapt);
     bool GetAdaptContentSize();
 
+    void ScrollTo(int x, int y);
```

Comment:
Good add, pretty nice JSBind automatically generates the JS/TS bindings, huh? :smile: